kratos/utilities/specifications_utilities.h

```diff
@@ -50,47 +50,18 @@ namespace
     template< class TContainerType>
     std::vector<std::string> GetDofsListFromGenericEntitiesSpecifications(const TContainerType& rContainer);
 
+    static std::unordered_map<std::string, GeometryData::KratosGeometryType> GenerateStringGeometryMap()
+    {
+        std::unordered_map<std::string, GeometryData::KratosGeometryType> my_map;
+        for (unsigned int i = 0; i < static_cast<unsigned int>(GeometryData::KratosGeometryType::NumberOfGeometryTypes); ++i) {
+            const auto type = static_cast<GeometryData::KratosGeometryType>(i);
+            my_map.insert({GeometryUtils::GetGeometryName(type), type});
+        }
+        return my_map;
```

Comment:
it would be cool if this could be computed at compile time, but not sure if C++17 can do that  (I think this requires C++20 `consteval`)

Would be cool if you can give it a try :)

Reply:
Yes, I check requires c++20